requirements.txt

```diff
@@ -1,11 +1,17 @@
-ansible==2.3.0.0
 appdirs==1.4.3
 arrow==0.10.0
+click==6.7
+emoji==0.4.5
+Flask==0.12.2
+itsdangerous==0.24
+Jinja2==2.9.6
+MarkupSafe==1.0
 cryptography==1.8.1
 packaging==16.8
 pyparsing==2.2.0
 python-dateutil==2.6.0
 requests==2.14.2
 schedule==0.4.2
+sh==1.12.13
 six==1.10.0
-emoji==0.4.5
+Werkzeug==0.12.2
```

Comment:
good eye.  i was testing some stuff and :hankey: 'd my local venv requirements.  let me fix..